ui-v2/app/services/feedback.js

```diff
@@ -16,10 +16,17 @@ export default Service.extend({
         });
       })
       .catch(e => {
-        get(this, 'flashMessages').add({
-          type: 'error',
-          message: error,
-        });
+        if (e.name === 'TransitionAborted') {
```

Comment:
Using promise success/fail for transition control flow must have felt so right at the time.